### DISCUSSION TITLE
Issue with ODF plot

### DISCUSSION THREAD

**Discussion by lilarou** [mtex-toolbox/mtex#2001] [Ask Anything]
Hi Mtex community,

I just installed the last version of MTEX (v6), before I used to work with v 5.8. 

I'm calculating ODF from EBSD data and based on grainArea for weight such as 
`CS = crystalSymmetry('6/mmm', [2.954 2.954 4.729],'X||a*', 'Y||b', 'Z||c*', 'mineral', 'Ti-Hex');
ebsd = EBSD.load(fname,CS,'interface','crc',...
  'convertSpatial2EulerReferenceFrame');
ebsdTi = ebsd('Ti-Hex');
grains= calcGrains(ebsdTi, 'threshold',2*degree, 'unitCell');
ori = grains.meanOrientation;
grainArea = grains.area;

odf1 = calcDensity(ori1,'weights',grainArea);

h = [Miller(0,0,0,1,CS) Miller(-1,1,0,0,CS) Miller(1,1,-2,0,CS)];
for i=1:length(h)
plotPDF(odf1,h(i),'smooth','parent',f.gca)
mtexColorbar
end`
The plotted pdf look like there is a 30° rotation. Before I never had this issue.
![image](https://github.com/mtex-toolbox/mtex/assets/123663516/dee843eb-29f9-48af-8862-a515249e8f7c)

Does anyone have an idea of why there is a 30° rotations ? 

Thank you

**Comment by kilir** [comment]
Hi,
have a look here for the big changes https://mtex-toolbox.github.io/changelog with mtex 6, one particular is the plotting convention which you likely need to adapt.

Hope that helps.

Cheers,
Rüdiger
